test/Makefile

```diff
@@ -0,0 +1,16 @@
+# Copyright 2017 The OpenPitrix Authors. All rights reserved.
+# Use of this source code is governed by a Apache license
+# that can be found in the LICENSE file.
+
+.PHONY: all
+all: generate test
+
+.PHONY: generate
+generate:
+	cat ../pkg/cmd/api/spec/*.swagger.json | jq --slurp 'reduce .[] as $$item ({}; . * $$item)' > api.swagger.json
+	swagger generate client -f api.swagger.json -A openpitrix
+	rm api.swagger.json
+
+.PHONY: test
+test:
+	go test
```

Comment:
if the local openpitrix server missing, the go test will failed.
maybe we should skip the test as default.
for developer, we hope `go test ./...` is as fast as `go build ./...`.
if we need do more test, we could enable the more test by command line flags.